autoload/nnn.vim

```diff
@@ -232,6 +232,14 @@ function! s:switch_back(opts, Cmd)
     if (type(a:Cmd) == v:t_string && a:Cmd != 'edit')
                 \ || (type(l:layout) != v:t_string
                 \ || (type(l:layout) == v:t_string && l:layout != 'enew'))
+        " delete the nnn window and buffer
+        if has('nvim')
+            if nvim_win_is_valid(l:term_wins.term.winhandle)
+                call nvim_win_close(l:term_wins.term.winhandle, v:false)
+            endif
+        else
+            call popup_close(l:term_wins.term.winhandle)
```

Comment:
Does `popup_close` work with non-popup window?

Reply:
That is a very good question, I should not have blindly copied that over. I am not as familiar with Vim's window API, do you know the right function to do so?

Reply:
`execute win_id2win(l:term_wins.term.winhandle) . 'close'` seems to work. Will make a new PR fixing this and also adding a try/catch around the whole thing.